src/uct/ib/dc/dc_mlx5_devx.c

```diff
@@ -29,11 +29,13 @@ ucs_status_t uct_dc_mlx5_iface_devx_create_dct(uct_dc_mlx5_iface_t *iface)
     dv.cq.in            = iface->super.super.super.cq[UCT_IB_DIR_RX];
     dv.cq.out           = &dvcq;
 
-    if (!UCT_RC_MLX5_TM_ENABLED(&iface->super)) {
+    if (iface->super.rx.srq.type != UCT_IB_MLX5_OBJ_TYPE_DEVX) {
```

Comment:
so also need to fix line 47?
intead, let's update uct_ib_mlx5_srq_init(), so iface->super.rx.srq.srq_num will always be correct (also in case of verbs srq)
so we can use iface->super.rx.srq.srq_num in lines 47 and 50